tabular/src/autogluon/tabular/models/tabular_nn/tabular_nn_model.py

```diff
@@ -782,6 +773,515 @@ def _get_default_stopping_metric(self):
         return self.eval_metric
 
 
+class TabularNeuralQuantileModel(TabularNeuralNetModel):
```

Comment:
Move to separate file

Reply:
moving to new file `tabuler_nn_torch.py`

Reply:
Doesn't look like this has been moved to separate file?

Reply:
Yes, I also move `TabularNeuralQuantileModel` into `tabuler_nn_torch.py`

Reply:
I don't think moving TabularNeuralQuantileModel into tabular_nn_torch.py will work because it seems like it'll break the lazy import?  
We need to ensure `try_import_torch()` is called before any `import torch` ever executes (including upon `import autogluon.tabular`).  So TabularNeuralQuantileModel may need to be in its own file.